src/styles/theme/index.ts

```diff
@@ -0,0 +1 @@
+export * from './theme'
```

Comment:
You can't re-export default in this way, same goes for all index.ts files in components
```suggestion
export { default } from './theme'
```